epictrack-web/src/components/workPlan/about/aboutDetails/WorkDetails.tsx

```diff
@@ -0,0 +1,133 @@
+import { Box, Divider, Grid } from "@mui/material";
+import { useContext, useEffect, useState } from "react";
+import { WorkplanContext } from "../../WorkPlanContext";
+import { ETCaption1, ETCaption2, GrayBox } from "../../../shared";
+import { Palette } from "../../../../styles/theme";
+import dayjs from "dayjs";
+import { MONTH_DAY_YEAR } from "../../../../constants/application-constant";
+
+const WorkDetails = () => {
+  const [phaseInfo, setPhaseInfo] = useState<any>({});
+  const { work, workPhases } = useContext(WorkplanContext);
+
+  const getCurrentAndNextPhaseInfo = () => {
```

Comment:
all good, just FYI, you can do something like const [phaseInfo, setPhaseInfo] = useState<any>(getCurrentAndNextPhaseInfo);

Reply:
I think that would be out of scope though?

Reply:
it's okay, just an FYI